test/integration/pairs_test.lua

```diff
@@ -296,3 +296,37 @@ add('test_le_condition_with_index', function(g)
     t.assert_equals(err, nil)
     t.assert_equals(objects, get_by_ids(customers, {1})) -- in age order
 end)
+
+add('test_negative_forst', function(g)
+    local customers = insert_customers(g,{
+        {
+            id = 1, name = "Elizabeth", last_name = "Jackson",
+            age = 12, city = "New York",
+        }, {
+            id = 2, name = "Mary", last_name = "Brown",
+            age = 46, city = "Los Angeles",
+        }, {
+            id = 3, name = "David", last_name = "Smith",
+            age = 33, city = "Los Angeles",
+        },
+    })
+
+    table.sort(customers, function(obj1, obj2) return obj1.id < obj2.id end)
+
+    -- negative first
+    local first = -10
+    t.assert_error_msg_contains("Negative first isn't allowed for pairs", function()
+        g.cluster.main_server.net_box:eval([[
+            local crud = require('crud')
+
+            local first = ...
+
+            local objects = {}
+            for _, object in crud.pairs('customers', nil, {first = first}) do
```

Comment:
could be replaced with simple one-liner. You just need to call `crud.pairs('customers', nil, {first = ...})` - without loop body